src/components/railBottom/controls/Controls.vue

```diff
@@ -143,6 +144,14 @@ export default ({
       this.yOffset = event.clientY - sourceElement.getBoundingClientRect().top;
     },
   },
+  async mounted() {
+    if (window.Cypress) {
```

Comment:
is this code necessary? It will be great if we can avoid references to the testing framework (cypress)

Reply:
This block code was removed on PR #1596